bot/src/main/kotlin/org/epilink/bot/rulebook/QuickHttp.kt

```diff
@@ -35,39 +104,25 @@ private val logger = LoggerFactory.getLogger("epilink.rulebooks.helpers")
  * @param url The URL for the GET request
  * @param basicAuth The Basic authentication credentials to use, or null to not use credentials at all
  * @param bearer The bearer to use in the Authorization header, or null to not do that
- * @param eagerAuthentication If the authentication information should be sent directly or only after the server sends
- * back an unauthorized error
+ * @param eagerAuthentication Since this method was rewritten to prevent thread leaking, this parameter doesn't change
+ * anything and is now always enabled
  */
 @Suppress("unused")
+@Deprecated("Reworked with better auth handling and custom return", ReplaceWith("httpGetJson(url, auth)"))
```

Comment:
ReplaceWith is a separate annotation no?

Reply:
It can be used like that apparently (it's actually how it is used in the default value)